heartbeat/docker-compose

```diff
@@ -148,12 +141,19 @@ docker_compose_status()
 {
 	# use docker-compose ps if YML found, otherwise try docker ps and kill containers
 	if [ -r "$DIR/$YML" ]; then
-		STAT_MSG=$(cd $DIR ; $COMMAND ps)
-		LNWTH=$(echo "$STAT_MSG" | expand | head -n1 | wc -c)
-		STATEPOS=$(echo "$STAT_MSG" | expand | head -n1 | egrep -o 'State.*$' | wc -c)
+
+		DKPS=$(cd $DIR; $COMMAND ps -q)
+		STAT_MSG=$(docker ps --no-trunc | grep -E $(echo "$DKPS" | tr '\n' '|') | expand)
+		LNWTH=$(echo "$STAT_MSG" | head -n1 | wc -c)
+		STATEPOS=$(echo "$STAT_MSG" | head -n1 | egrep -o 'STATUS.*$' | wc -c)
 		OFFSET=$(($LNWTH-$STATEPOS+1))
-		PSNU=$(echo "$STAT_MSG" | grep "^${SVC}_" | wc -l)
-		UPNU=$(echo "$STAT_MSG" | grep "^${SVC}_" | expand | cut -c ${OFFSET}- | awk '{print $1}' | grep -w 'Up' | wc -l)
+
+		if [ "$DKPS" ]; then
```

Comment:
This should be `if [ -n "$DKPS" ]; then` (not zero length).

Reply:
I think it should be okay without the -n operator, according to the man(1) sh:
            string             string has non-zero length.
            -n string          string is not empty.
Anyway, I will put the -n to the code at the next commit. Thanks for your comment.